k8s/k8s.bzl

```diff
@@ -44,4 +44,4 @@ py_library(
   )
 
   # WORKSPACE target to configure the kubectl tool
-  kubectl_configure(name = "local_k8s_config")
+  kubectl_build(name = "local_k8s_config")
```

Comment:
Change 'local_k8s_config' to 'k8s_config' or 'k8s_build'. Also skip kubectl_build if the repository key exists in native.existing_rules().keys(). See https://github.com/bazelbuild/rules_docker/blob/9527234ef0b5a57bce93be524cb56d7ab1a85ea3/container/container.bzl#L227 for how we did this in rules_docker

Reply:
done, thanks for the reminders